src/views/Player.vue

```diff
@@ -38,6 +38,10 @@ var interval_time = 50;
 var upload_interval = 45000;
 var timeout = null;
 
+// wait this much time (secs) then show error page
+// if browser is not supported
+var browser_check_time = 5;
+
```

Comment:
Removed hardcoding, used a variabled. We'll wait for 5 seconds, and if the seek bar does not move till then, we'll show error page.

[x] - What if net speed is so slow that the video didn't load for 5 seconds? Can we detect slow internet and change this variable accordingly?

Reply:
change to 10